var/spack/repos/builtin/packages/fckit/package.py

```diff
@@ -50,6 +53,9 @@ def cmake_args(self):
 
         if '~shared' in self.spec:
             args.append('-DBUILD_SHARED_LIBS=OFF')
+        
+        if 'finalize_ddts=auto' not in self.spec:
```

Comment:
I think the Spack team-preferred syntax is `if not self.spec.satisfies("finalize_ddts=auto"):` but otherwise this looks fine. It's not like we don't use `in self.spec` in a million other places...